### PR TITLE
add hack time session format

### PR DESCRIPTION
**Pull request by flyingzumwalt**
@diasdavid take a look & merge if it looks good to you!

### CODE REVIEW

File: sessions/README.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ You can propose any session format that you think will work for achieving your o
 - [Project Update Lightning Talks](./lightning-talks): Rapid-fire information-dense project updates
 - [Deep Dive](./deep-dive): Deep technical design sessions
 - [Poster Making](./poster-making): Work in small groups to create posters and other documentation, then give a one-minute presentation of your work.
-- **Choose your own format!** if you want to propose your own format, follow the instructions in the next few steps.
+- [Hack Time](./hack-time): In pairs or small teams to tackle work units explicitly focused on code. Aim for lots of pair programming.
```
Comment (daviddias) [inline sessions/README.md]: I would dare to make this time _pairs_ only

File: sessions/hack-time/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- **Outcomes:** Commit improvements & fixes and review or merge outstanding PRs related to important work packages.
+
+## Details
+
+
+# Proposed Hack Time Work Packages
+
+**Instructions**
+- **If you want to propose something to hack on**, add it here (via a Pull Request). Optionally use the [template](../_template.md) to provide a fuller description of the session.
+- **At the start of Hack Time** everyone will choose work packages using post-it notes: All of the work packages will be listed on pieces of paper on a wall. Each person gets one post-it note, puts their name on it, and attaches it to a work package. _Maximum of 3 people per work package._ If the work package is maxed out, you need to choose another one or negotiate swapping with someone else in the group.
```
Comment (daviddias) [inline sessions/hack-time/README.md]: This is still saying `work packages` and describing max of 3 people. Hacking in trios is hard and inefficient. Effective pair programming is 2 only

**Comment by flyingzumwalt** [comment]
updated. please give it another glance @diasdavid

**Comment by flyingzumwalt** [comment]
(oops I missed some references to small teams and trios. updated again)

**Pull request merged**